test/rendering.test.js

```diff
@@ -0,0 +1,351 @@
+import { beforeAll, describe, expect, it } from 'vitest';
+import { mount } from "@vue/test-utils";
```

Comment:
Gibt es einen speziellen Grund warum wir nicht die [Vue Testing Library](https://testing-library.com/docs/vue-testing-library/intro/) benutzen? Die setzt ja auf `@vue/test-utils` auf und ist etwas DOM-näher in den Tests. Ist glaube am Ende des Tages für dieses Projekt relativ egal. Also bin hier leidenschaftslos. War jetzt nur neugierig.

Reply:
Der wichtigste Grund ist, dass ich es nicht kannte :D

Ich habe `@vue/test-utils` schon für eine Bibliothek, die ich maintaine benutzt und war geflasht, wie gut das ging. Danach habe ich nicht weiter nach Testlibs geschaut.

Schaue ich mir auf jeden Fall mal an.